avalon/tools/creator/app.py

```diff
@@ -301,15 +302,29 @@ def _on_data_changed(self):
             # Get plugin and family
             plugin = item.data(PluginRole)
             family = plugin.family.rsplit(".", 1)[-1]
+            regex = "{}*".format(family)
+            existed_subset_split = family
+
+            if family in self.taskSubsetFamilies:
+                task = io.Session.get('AVALON_TASK', '')
+                sanitized_task = re.sub('[^0-9a-zA-Z]+', '', task)
```

Comment:
we can replace this with `\d`

Reply:
No it cannot. This way with \d it is not getting anything else but numbers.. Didn't find shorter equivalent to this.. Closest was `\D` but it is not getting `_` so I will keep this as it is..

Reply:
I think `[^\W_]+` might work but this is maybe more readable